tests/testthat/test-getLargeScaleCharacteristics.R

```diff
@@ -42,17 +44,6 @@ test_that("check getLargeScaleCharacteristics inputs checks", {
     tablesToCharacterize = c("drug_exposure")
   ))
 
```

Comment:
why these tests were deleted?

Reply:
> why these tests were deleted?

cuz I edited the mock function to contain all cdm tables now, previously measurement and device_exposure tables are not in cdm without user specify

Reply:
Can you then add 
cdm$drug_exposure <- NULL
and see that the error appears?